get_webpack_manifest.py

```diff
@@ -0,0 +1,23 @@
+import json
+import os
+import warnings
+from django.conf import settings
+
+
+def get_webpack_manifest(path=None):
+    manifest = {}
+
+    if not path:
+        path = os.path.join('webpack/_build/manifest.json')
+    if not os.path.exists(path):
+        if not settings.UNIT_TESTING and settings.DEBUG:
+            warnings.warn("\x1b[33;20m"  # yellow color
```

Comment:
Should this be a hard error rather than a warning? I'm assuming that if the manifest isn't present, pages depending on webpack will break.